benchmarks/time_forest_partition.cxx

```diff
@@ -337,8 +345,10 @@ t8_time_forest_create_cmesh (const char *msh_file, int mesh_dim,
      * among all ranks */
     cmesh = t8_cmesh_load_and_distribute (cmesh_file, num_files, comm,
                                           T8_LOAD_STRIDE, stride);
-    /* Partition only if more than 1 input file */
-    partition = num_files > 1;
+    /* Partition only if more than 1 input file and no curved option*/
+    if (num_files > 1 && !use_occ) {
+      partition = num_files > 1;
+    }
```

Comment:
This bit of code is only executed, when no `msh` file is specified, and we only support curved geometries for `msh` files. Therefore, this change here is not necessary. But we should abort, if the user uses the `use_occ` option but does not provide an `msh` file along with it